yolort/relay/ir_visualizer.py

```diff
@@ -11,12 +11,13 @@
 """
 from collections import OrderedDict
 
-try:
+import yolort.utils.dependency as _dependency
+
+if _dependency.is_module_available("graphviz"):
     from graphviz import Digraph
-except ImportError:
-    Digraph = None
 
 
+@_dependency.requires_module("graphviz")
```

Comment:
Hi @mattpopovich , as mentioned in https://github.com/zhiqwang/yolov5-rt-stack/pull/366#discussion_r829036755, we suggest to move this decorator before the following `self.render()` method

https://github.com/zhiqwang/yolov5-rt-stack/blob/bf05b4b510b2a41ce4d8975b9b1be33784d29450/yolort/relay/ir_visualizer.py#L50

And the reason is same, we wanna make the Class TorchScriptVisualizer more applicable/general in the future even if we don't install the `graphviz` package here.